public/app/features/iam/api/scripts/generate-rtk-apis.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * To generate query library k8s APIs, run:
+ * `yarn process-specs && npx rtk-query-codegen-openapi ./public/app/features/iam/api/scripts/generate-rtk-apis.ts` from the root of the repo
+ */
+
+import { ConfigFile } from '@rtk-query/codegen-openapi';
+import { accessSync } from 'fs';
+
+const schemaFile = '../../../../../../data/openapi/iam.grafana.app-v0alpha1.json';
+
+try {
+  // Check we have the OpenAPI before generating query library RTK APIs,
+  // as this is currently a manual process
+  accessSync(schemaFile);
+} catch (e) {
+  console.error('\nCould not find OpenAPI definition.\n');
+  console.error(
+    'Please run go test pkg/tests/apis/openapi_test.go to generate the OpenAPI definition, then try running this script again.\n'
+  );
+  throw e;
+}
```

Comment:
I _think_ RTK generator will throw an error anyways if the schema file doesn't exist, so not sure we need this.

Reply:
yeah i think it will, but i prefer leaving this in for now since it gives some specific instructions as to what to do. i expect this will change when we pull all these scripts together into one anyway 🤔